internal/website/content/howto/docstore/actions.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Actions"
+date: 2019-06-11T10:11:59-04:00
+draft: true
+weight: 2
+---
+
+In Docstore, you use actions to read, modify and write documents. You can
+execute a single action, or run multiple actions together in an _action list_.
+
+<!--more-->
+
+## Representing Documents
+
+We'll use a collection with documents represented by this Go struct:
+```
+type Player struct {
+	Name             string
+	Score            int
+	DocstoreRevision interface{}
+}
+```
+
+We recommend using structs for documents because they impose some structure on
+your data, but Docstore also accepts `map[string]interface{}` values. See
+[the `docstore` package
+documentation](https://godoc.org/gocloud.dev/internal/docstore#hdr-Documents)
+for more information.
+
+The `DocstoreRevision` field holds information about the latest revision of the
+document. We discuss it [below]({{< ref "#docrev" >}}).
+<!-- TODO(jba): make sure the above link works -->
```

Comment:
It should. (Not sure what your dev workflow is, but you can check the site by running `hugo serve -D` from the `internal/website` directory.)